app/lib/client_sorter.rb

```diff
@@ -27,7 +27,6 @@ def active_filters
 
   def filtered_and_sorted_clients(default_order: nil)
     @default_order = default_order
-    setup_sortable_client unless @filters.present?
```

Comment:
This line appears to never be possible to hit, and if it ever were hit it would throw an exception. I removed it while I was in there.